schema.graphql

```diff
@@ -460,6 +499,21 @@ type MatchQuestion {
   Information about the bonus that was read for this question. Must be omitted if `bonus_points` is present; may be omitted if no bonus was read for this question, but should not be omitted merely because no points were earned on the bonus.
   """
   bonus: MatchQuestionBonus
+
+  """
+  The time at which the moderator began reading the bonus. An ISO 8601-formatted string, with time zone, in the tournament's local time zone.
+  """
+  bonus_start_time: Date
+
+  """
+  Information about the bonus question used as a replacement in this cycle, if any. Omit if the bonus was not replaced. Multiple replacement bonuses are not currently supported.
+  """
+  replacement_bonus_question: Question
+
+  """
+  The `MatchTeam`s who were eligible to answer the replacement bonus. Any `MatchTeam`s in this match but not in this field should be understood to have been eligible to answer the original `bonus.question`.
+  """
+  replacement_bonus_question_match_teams: [MatchTeam]
```

Comment:
Is this something weird involving bouncebacks?

Reply:
Yeah. In formats with bouncebacks, depending on the circumstances that led to the use of a replacement bonus, one or both teams may be eligible to answer a replacement bonus.

Don't ask me how bouncebacks work in formats with more than two teams :-)